pgjdbc/src/test/java/org/postgresql/test/jdbc3/Jdbc3CallableStatementTest.java

```diff
@@ -1008,4 +1011,22 @@ public void testSum() throws SQLException {
     assertEquals("2+3 should be 5 when executed via {?= call mysum(?, ?)}", 5, cs.getInt(1));
   }
 
+  @Test
+  public void testNoParametersWithParentheses() throws SQLException {
+    assumeCallableStatementsSupported();
+    CallableStatement cs = con.prepareCall("{?= call mynoparams()}");
+    cs.registerOutParameter(1, Types.INTEGER);
+    cs.execute();
+    assertEquals("{?= call mynoparam()} should return zero, but did not.", 0, cs.getInt(1));
+  }
+
+  @Test
+  public void testNoParametersWithoutParentheses() throws SQLException {
+    assumeCallableStatementsSupported();
+    CallableStatement cs = con.prepareCall("{?= call mynoparams}");
```

Comment:
Can you please add a couple of tests for calling a procedure?
I mean something around `{ call mynoparams}` and `{ call mynoparams() }`.

`@Parameterized` test might be great way to explore different combinations of whitespace, braces, etc